src/plugins/mavlink_ftp/include/plugins/mavlink_ftp/mavlink_ftp.h

```diff
@@ -0,0 +1,189 @@
+#pragma once
+
+#include <vector>
+#include <memory>
+#include <string>
+#include <functional>
+
+// This plugin provides/includes the mavlink 2.0 header files.
+#include "mavlink/v2.0/common/mavlink.h"
+#include "plugin_base.h"
+
+namespace mavsdk {
+
+class System;
+class MavlinkFTPImpl;
+
+/**
+ * @brief The MavlinkFTP class provides MAVLink FTP functionality
```

Comment:
Maybe add a comment one step further back.

```suggestion
 * @brief The MavlinkFTP class provides file uploading and downloading using MAVLink FTP.
```